projects/js-packages/charts/src/components/bar-chart/bar-chart.tsx

```diff
@@ -28,30 +23,48 @@ const BarChart: FC< BarChartProps > = ( {
 } ) => {
 	const theme = useChartTheme();
 	const { tooltipOpen, tooltipLeft, tooltipTop, tooltipData, hideTooltip, showTooltip } =
-		useTooltip< DataPoint >();
+		useTooltip< { value: number; xLabel: string; yLabel: string; seriesIndex: number } >();
```

Comment:
This type is kind of long, should we extract it above with the interface?

Reply:
Good point. I'll add it to the next set of changes.